docs/history/changelog.md

```diff
@@ -237,6 +237,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * [#1067](https://github.com/open-contracting/standard/pull/1067) `Publisher.name`, to indicate that it is the organization or department responsible for publishing the OCDS version of the data.
 
 * [#1450](https://github.com/open-contracting/standard/pull/1450) Replace a repeated example in schema/merging/ with a link to guidance/build/merging/.
+* [#1450](https://github.com/open-contracting/standard/pull/1472) Remove planning reference from the versioned release example.
```

Comment:
I don't think we need this entry. I assume the example had been updated, but we just forgot to change the text here. We can add 1472 to that other entry (haven't checked), or we can just not mention 1472.

Reply:
Apparently, the JSON file never contained a planning stage https://github.com/open-contracting/standard/commit/42b42089ca968e004359c41cd5a743f7cf0b051f, and the text was like that since the begging of time https://github.com/open-contracting/standard/commit/b83f1be62fe2a65c511fa2c6b9415d0c1a9f9609. So I will remove this entry from the changelog